tests/test_imports.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+
+def test___all__(pytester: pytest.Pytester) -> None:
+    """See https://github.com/aio-libs/aiohttp/issues/6197"""
+    pytester.makepyfile(
+        test_a="""
+            from aiohttp import *
```

Comment:
Suppose this could be more accurate with:
```suggestion
            from aiohttp import *
            assert 'GunicornWebWorker' in globals()
```